### PR TITLE
Removes word "beta" from docs

### PR DESCRIPTION
**Pull request by johnnymatthews**
Closes ipfs/docs#510

### CODE REVIEW

**Comment by johnnymatthews** [comment]
> [I'll update any image assets related to this too.](https://github.com/ipfs/docs/issues/510#issuecomment-628776538)
@cwaring 

Feel free to make your image asset updates in this branch, if you've not already updated things. If you have then we can merge this in whenever.

**Comment by hugomrdias** [comment]
@johnnymatthews is this ready to merge ? we can update assests later

**Comment by johnnymatthews** [comment]
Yeahp, it's ready to go in whenever.

File: docs/community/contribute/grammar-formatting-and-style.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ How the Markdown syntax looks, and code formatting rules to follow.
 
 ### Syntax
 
-The IPFS Docs project follows the _GitHub Flavoured Markdown_ syntax for markdown. This way, all articles display properly within GitHub itself. This gives readers the option to view articles on [the docs website](https://docs-beta.ipfs.io) or [its GitHub repo](https://github.com/ipfs/ipfs-docs-v2).
+The IPFS Docs project follows the _GitHub Flavoured Markdown_ syntax for markdown. This way, all articles display properly within GitHub itself. This gives readers the option to view articles on [the docs website](https://docs.ipfs.io) or [its GitHub repo](https://github.com/ipfs/ipfs-docs-v2).
```
Comment (hugomrdias) [inline docs/community/contribute/grammar-formatting-and-style.md:43]:
```suggestion
The IPFS Docs project follows the _GitHub Flavoured Markdown_ syntax for markdown. This way, all articles display properly within GitHub itself. This gives readers the option to view articles on [the docs website](https://docs.ipfs.io) or [its GitHub repo](https://github.com/ipfs/ipfs-docs).
```

**Pull request merged**